{
  "comments": [
    {
      "key": {
        "uuid": "9a245df4_76c107e4",
        "filename": "pcep/impl/src/main/java/org/opendaylight/protocol/pcep/impl/PCEPSessionImpl.java",
        "patchSetId": 9
      },
      "lineNbr": 367,
      "author": {
        "id": 7569
      },
      "writtenOn": "2017-10-09T21:27:28Z",
      "side": 1,
      "message": "Since exceptionCaught() is an overridden method from netty, for ease of readability, it might be better to create a separate method for exception handling e.g. handleException() and use it here as well as in exceptionCaught()",
      "revId": "0cd5cbf51b1f886c5b52541e31b0056df698e86d",
      "serverId": "7fc14799-209e-464c-9743-7a06c2c21a81",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "9a245df4_96bc7b6e",
        "filename": "pcep/impl/src/main/java/org/opendaylight/protocol/pcep/impl/PCEPSessionImpl.java",
        "patchSetId": 9
      },
      "lineNbr": 427,
      "author": {
        "id": 7569
      },
      "writtenOn": "2017-10-09T21:27:28Z",
      "side": 1,
      "message": "final missing from method arguments",
      "revId": "0cd5cbf51b1f886c5b52541e31b0056df698e86d",
      "serverId": "7fc14799-209e-464c-9743-7a06c2c21a81",
      "unresolved": false
    }
  ]
}